src/scss/components/dropdown-select/_config.scss

```diff
@@ -4,6 +4,10 @@
 
 /// ========== MIXINS ========== ///
 @import "mixins";
+@import "../form/mixins/fieldName";
+@import "./mixins/SelectControl";
+@import "./mixins/SelectFacade";
+
 
 // LESS               SCSS
```

Comment:
Please move this mixin into it's own file -> `components/dropdown-select/mixins/SelectReset.scss`.